src/main/java/com/google/api/codegen/transformer/SampleTransformer.java

```diff
@@ -133,8 +131,8 @@ public void generateSamples(
     for (CallingForm form : callingForms) {
       MethodConfig methodConfig = context.getMethodConfig();
 
-      Set<SampleValueSet> matchingValueSets;
-      matchingValueSets = methodConfig.getSampleSpec().getMatchingValueSets(form, sampleType);
+      List<SampleValueSet> matchingValues =
```

Comment:
This variable should still be `matchingValueSets`, since it holds a collection of `SampleValueSet`s, each of which holds individual, related values.

Reply:
Done